c/tskit/tables.h

```diff
@@ -62,6 +64,8 @@ Sizes in tskit are defined by the ``tsk_size_t`` type.
 @endrst
 */
 typedef uint32_t tsk_size_t;
+#define TSK_MAX_SIZE (tsk_size_t) UINT32_MAX
```

Comment:
Maybe put in some extra brackets, to be save, ``((tsk_size_t) UINT32_MAX)``

Reply:
I've removed the cast - not sure why it was there.